### ISSUE TITLE
Case Sensitivity

### ISSUE THREAD

**Issue by Choco31415** [tjvr/scratchblocks#157]
Blob8108, I asked the international Scratch Wiki and they do not want case sensitivity because of the potential problems it can cause.

**Comment by tjvr** [comment]
What problems are those? :)

**Comment by Choco31415** [comment]
Blocks turning red because you just happen to use a capital "M" somewhere instead of a regular "m".

**Comment by tjvr** [comment]
That is how it works, yes. You're not really giving me a reason why it's bad :)

**Comment by Choco31415** [comment]
Having case sensitivity would break several existing scripts in the international Scratch Wikis. I have confirmed that the international wikis do use incorrectly capitalized blocks. Ex: http://scratch-dach.info/wiki/Ändere_Lautstärke_um_()_(Block) Although there are not many such pages, it would not be nice to have to find all of them and fix them.

( That and I found a German block that doesn't work in v.3. I used the same spelling & capitalization as the translation file: http://i.cubeupload.com/jbXCAX.png )

On the flip side, the Wikis generally do not care about case sensitivity, and as such, case sensitivity does not add a substantial benefit.
